src/components/Main/Mitigation/MitigationTable.tsx

```diff
@@ -135,22 +135,20 @@ function MitigationIntervalComponent({
             name={`containment.mitigationIntervals[${index}].name`}
             type="text"
           />
-          <div className="item-date-range-value-group">
-            <MitigationDatePicker
-              identifier={`containment.mitigationIntervals[${index}].timeRange`}
-              value={interval.timeRange}
-              allowPast
-            />
-            <FastField
-              className={`form-control item-value ${valueError ? 'border-danger' : ''}`}
-              id={`containment.mitigationIntervals[${index}].mitigationValue`}
-              name={`containment.mitigationIntervals[${index}].mitigationValue`}
-              type="number"
-              step={1}
-              min={0}
-              max={100}
-            />
-          </div>
+          <MitigationDatePicker
```

Comment:
Hi @natkaratkova, thanks for working on this and I apologize for this horrendous CSS that I created.
One difference between the current behavior and your PR is that the interval switches to 3 rows when the space for the mitigation table is < 580px. In the current implementation the date range and value are kept in a row. Is there a way to get that behavior in your PR?

Reply:
hi @gj262, no worries, your CSS is very reasonable :) Thanks for the feedback, I see. I'll fix this

Reply:
I continued without a row (to stay within stylelint specificity limits). Hope hasn't broken anything. The "very narrow" view has changed a little but it fixed some date picker overlap <img width="817" alt="Screen Shot 2020-04-16 at 05 21 20" src="https://user-images.githubusercontent.com/2752193/79414494-afdac100-7fa2-11ea-9fa4-cd0987f79f5e.png">